### PR TITLE
Laravel 6 support

### PR DESCRIPTION
**Pull request by EmanueleCoppola**

### CODE REVIEW

**Comment by kylekatarnls** [comment]
Great! Thanks a lot.

**Comment by kylekatarnls** [comment]
Tests won't pass on Travis-CI as PHP < 5.6 now are on archived distributions on Travis. I will have to upgrade the .travis.yml file, I will add builds for Laravel 6 too, then I will merge this.

File: .travis.yml [hunk 1]
```diff
@@ -5,26 +5,6 @@ git:
 
 matrix:
   include:
-    - php: 5.4
```
Comment (EmanueleCoppola) [inline .travis.yml:8]: Sorry, I thought you wanted to remove that because of PHP support end. @kylekatarnls

Comment (kylekatarnls) [reply .travis.yml:8]: End of support version does not mean PHP will suddently stop working if you use PHP 5.4 and if we don't test it we won't know if we're still compatible with it (or rather we will probably miss incompatibilites). So before any PHP version support drop in the code or in testing environnent, any PHP software should first increase the minimum requirement in its composer.json to reflect the reality, then bump our own version number (at least a minor bump). Then I will not remove PHP 5 current range as long as we don't need a change big enough to rewrite the whole thing up to the PHP 7 standards.

**Comment by kylekatarnls** [comment]
Build time is so painful I also add some optimization in this PR to make them run faster.

**Comment by kylekatarnls** [comment]
Laravel 6 compatible version released:
https://github.com/BKWLD/laravel-pug/releases/tag/1.8.0

**Pull request merged**